docs/manual/en/introduction/How-to-use-WebGL2.html

```diff
@@ -48,9 +48,14 @@ <h2>Workflow</h2>
 		automatically convert the built-in material's shader code to GLSL ES 3.00.
 	</p>
 
+	<p>
+		Since you are manually creating the WebGL 2 rendering context, you also have to pass in all necessary context attributes.
+		Note: It's not possible to modify these attributes after the context has been created, so passing them to the WebGLRenderer won't have any effect.
+	</p>
+
 	<code>
 var canvas = document.createElement( 'canvas' );
-var context = canvas.getContext( 'webgl2' );
+var context = canvas.getContext( 'webgl2', { antialias: false } );
```

Comment:
I would remove this change. It's a minimal code example and `{ antialias: false }` does not make sense here. It's set in `webgl2_multisampled_renderbuffers` because multisampled renderbuffers (offscreen MSAA) should be demonstrated.

Reply:
Is there a contextAttribute that makes sense here? I'd just like see a demonstration on how one should do this the right way.

What if I change it to `{ alpha: false }`?

Reply:
Sounds good.

Reply:
Changed to `{ alpha: false }`